CHANGES.rst

```diff
@@ -30,6 +30,9 @@ Bug Fixes
 Cubeviz
 ^^^^^^^
 
+- Subsets from the spectrum viewer are now returned as SpectralRegion objects, and
```

Comment:
The first entry is technically API change, not just a bug fix. I propose we split this entry into different "API change" and "Bug fix" entries.